### PR TITLE
LPS-20304

### PR DESCRIPTION
**Pull request by MohammedElkhaldi**

### CODE REVIEW

**Comment by pedroamarall** [comment]
ci:test:sf

**Comment by Youness-ELMO** [comment]
@MohammedElkhaldi please fix the formatting! 
1- ant format-source-local-changes
2- ant format-source-current-branch

File: portal-web/test/functional/com/liferay/portalweb/tests/mytest/FormTests.testcase [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+		TestForm.clickOnForm();
+	}
+
+	@description = "Create a Form with a Text field and a Numeric field "
+	@priority = 5
+	test CheckPortlet {
+		ProductMenu.gotoPortlet(
+			category = "Content & Data",
+			portlet = "Forms");
+	}
```
Comment (Youness-ELMO) [inline portal-web/test/functional/com/liferay/portalweb/tests/mytest/FormTests.testcase:30]: use the BDD (Given. When, Then)

File: portal-web/test/functional/com/liferay/portalweb/tests/mytest/MyBlogTests.testcase [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+	@priority = 5
+	test CanCreateBlog {
+		task ("Given the Blogs Portlet is added to the page") {
+			JSONBlog.addEntry(
+				entryContent = "Extra Blog",
+				entryTitle = "Extra Blog");
+		}
+
+		
+	}
```
Comment (Youness-ELMO) [inline portal-web/test/functional/com/liferay/portalweb/tests/mytest/MyBlogTests.testcase:34]: 1. CanCreateBlog can create successfully or fail
2. and for this test u need to crate the blog with the UI first then use JSON on Edit, Delete and Search 
3. use the BDD (u need here When and Then)

File: portal-web/test/functional/com/liferay/portalweb/tests/mytest/MyBlogTests.testcase [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+				entryTitle = "Blog Zindigi");
+			}
+				BlogsNavigator.openBlogsAdmin(siteURLKey = "guest");
+
+		task ("Then Delete the Blog contain this Title") {
+			LexiconEntry.gotoEntryMenuItem(
+				menuItem = "Delete",
+				rowEntry = "Blog Zindigi");
+		}		
+	}
```
Comment (Youness-ELMO) [inline portal-web/test/functional/com/liferay/portalweb/tests/mytest/MyBlogTests.testcase:52]: 1. where is When (BBD)
2. and the validation for the delete (AssertElementNotPresent)

File: portal-web/test/functional/com/liferay/portalweb/tests/mytest/MyBlogTests.testcase [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+				rowEntry = "Blog Zindigi");
+		}
+
+		task ("Then Edit the Blog") {
+
+			BlogsEntry.editTitle(
+				entryTitle = "Blog Zindigi",
+				entryTitleEdit = "Blog Malmali");
+		}		
+	}
```
Comment (Youness-ELMO) [inline portal-web/test/functional/com/liferay/portalweb/tests/mytest/MyBlogTests.testcase:74]: the same here!

File: portal-web/test/functional/com/liferay/portalweb/tests/mytest/MyBlogTests.testcase [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+			JSONBlog.addEntry(
+				entryContent = "Blog Content Liferay",
+				entryTitle = "Blog Liferay");
+		}
+
+		task ("Then can Search for Blogs") {
+			BlogsEntry.searchCP(entrySearchTerm = "Blog Liferay");
+		}
+	}
+
```
Comment (Youness-ELMO) [inline portal-web/test/functional/com/liferay/portalweb/tests/mytest/MyBlogTests.testcase:90]: also here

File: portal-web/test/functional/com/liferay/portalweb/tests/mytest/MyTestFile.testcase [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+	@description = "Set Screen size to custom and validate values"
+	@priority = 5
+	test SetScreenSizeAndValidateValues {
+		MobileDevice.previewPG(
+			applyCustomSize = "true",
+			deviceType = "custom",
+			height = 700,
+			width = 700);
+	}
+
```
Comment (Youness-ELMO) [inline portal-web/test/functional/com/liferay/portalweb/tests/mytest/MyTestFile.testcase:33]: use BBD

**Pull request closed without merging**